fotoobo/helpers/config.py

```diff
@@ -29,7 +29,6 @@ class Config:
     audit_logging: Union[Dict[str, Any], None] = None
     no_logo: bool = False
     cli_info: Dict[str, Any] = field(default_factory=dict)
-    ssl_verify: Union[str, bool] = True
```

Comment:
Where does this removal come from? Was this dead code before? If not we should not update this with a patch release...